index.html

```diff
@@ -1,16 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <script src="script.js" defer></script>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
     <title>Quote Generator</title>
 </head>
+
 <body>
+    <div class="container">
+        <header> Quote of the day! </header>
+        <div class="content">
+            <div class="quoteArea">
+                <i class="fas fa-quote-left"></i>
+
+                <p class="quote"> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sequi, ea? </p>
+                <i class="fas fa-quote-right"></i>
+            </div>
+
+            <div class="author">
+
+                
+                <span class="nameOfAuthor">Nikola Tesla</span>
+            </div>
+
+        </div>
+        <div class="buttonsContainer">
+            <ul>
+                <li class="copy"> <i class="fas fa-copy"></i> </li>
+            </ul>
+            <button class="btn">Get a new Quote</button>
```

Comment:
You can correct here text in the button, you can write "New quote" instead of "get a new quote".